content/the-ember-times-issue-189.md

```diff
@@ -66,15 +68,13 @@ If this is something you are interested in, please read and discuss the [Asset I
 
 ---
 
-## [🐹 4. Section title in sentence case](section-url)
+## [🏛️ EmberFest 2021 - best of both worlds](https://emberfest.eu)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Next week after a year of absence because of COVID [EmberFest](https://emberfest.eu) is back. On the 30th of September and the 1st of October Emberistas will gather **digitally** and **analogue** in Rome to hear awesome Ember-talks.
```

Comment:
Content looks good. I updated a few punctuation marks to help with understanding each sentence.

```suggestion
Next week, after a year of absence because of COVID, [EmberFest](https://emberfest.eu) is back. On the 30th of September and the 1st of October, Emberistas will gather **digitally** and **in person** in Rome to hear awesome Ember talks.
```